### PR TITLE
new discount entries 

### PR DESCRIPTION
**Pull request by dipt464**
new discount entries added. please review this.

### CODE REVIEW

File: FeaturedProducts.md [hunk 1]
```diff
@@ -27,4 +27,8 @@
 ## Monday deals (**only for our loyal customers)
 - indian pizza: pizza made with an indian twist  (Price:$6)
 - italian pizza: pizza made with an italian twist  (Price:$6)
- 
\ No newline at end of file
+ 
+ ## weekend deal 
+- biryani at 50% off
```
Comment (invisibleparticle) [inline FeaturedProducts.md]: Make 'b' in biryani in uppercase.

**Review by invisibleparticle — CHANGES_REQUESTED** [review]

File: FeaturedProducts.md [hunk 2]
```diff
@@ -27,4 +27,8 @@
 ## Monday deals (**only for our loyal customers)
 - indian pizza: pizza made with an indian twist  (Price:$6)
```
Comment (invisibleparticle) [inline FeaturedProducts.md]: Correct the case of i

File: FeaturedProducts.md [hunk 3]
```diff
@@ -27,4 +27,8 @@
...
 - italian pizza: pizza made with an italian twist  (Price:$6)
```
Comment (invisibleparticle) [inline FeaturedProducts.md]: make i in italian in uppercase.

**Review by invisibleparticle — CHANGES_REQUESTED** [review]

File: FeaturedProducts.md (hunk 2 above)
Comment (dipt464) [reply FeaturedProducts.md]: changes done

File: FeaturedProducts.md (hunk 3 above)
Comment (dipt464) [reply FeaturedProducts.md]: changes done

**Review by invisibleparticle — APPROVED** [review]
All look good now. Thanks!

**Review by dipt464 — COMMENTED** [review]
I have done self review.

**Pull request merged**